scripts/benchmark_inference.py

```diff
@@ -42,6 +44,14 @@
 #         71.45 ms per token
 
 
+# Basic process setup for making results deterministic
+SEED = 42
+random.seed(SEED)
+torch.manual_seed(SEED)  # pytorch random seed
+np.random.seed(SEED)  # numpy random seed
+torch.backends.cudnn.benchmark = False
+torch.backends.cudnn.deterministic = True
```

Comment:
We should make this optional since determinism can impair performance. I'd suggest re-using the flag in the inference.py script:

https://github.com/foundation-model-stack/foundation-model-stack/blob/main/scripts/inference.py#L70

Also I believe `torch.backends.cudnn.deterministic`  only affects convolution operations, whereas I believe `torch.use_deterministic_algorithms(True)` would affect all operators.

Reply:
Thanks for your comments. I updated the PR.